### PR TITLE
Added rework dl wrapup blog post

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: _posts/2019-09-27-rework-dl-summit.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+type: text
+---
+
+## Conference Summary
+
+I’m back at the [RE•WORK DL summit](https://www.re-work.co/events/deep-learning-summit-london-2019) again this year to catch up with some of the hottest research in Deep Learning (DL). I've shared a reading list at the end of this article that is related to the main track sessions I attended.
+
+Of particular interest to me was a talk by [Richard Turner](http://www.eng.cam.ac.uk/profiles/ret26) that uses [adaptive networks](https://arxiv.org/abs/1906.07697) to allow continuous training with new data instead of having to re-train from scratch.
+
+Reinforcement Learning was heavily featured this time as researchers coax DL algorithms to paint, control robots, drive, and play starcraft. There was also a variety of talks related to autonomous driving that range from making sense of sensor data to controlling the car itself.
```
Comment (drj11) [inline _posts/2019-09-27-rework-dl-summit.md:24]: We have to have a picture of starcraft. More seriously: for the social media card (the preview when you send a link via Twitter or Slack or WhatsApp or similar), it's a good idea to include a picture.

File: _posts/2019-09-27-rework-dl-summit.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+In the talks, the problem of data inefficiency is an issue that came up often, for example Alphastar AI agents are trained for over 200 gaming years to be able to compete with the best players. Another important issue that remains unsolved, especially for safty critical systems, is how to ensure that the algorithms behave sensibly in all situations, even in the rare cases where there's little or no available training data. 
+
+[Prof. Neil Lawrance's](http://inverseprobability.com) talk on [Machine Learning Systems Design](http://inverseprobability.com/2018/11/05/the-3ds-of-machine-learning-systems-design) highlights these problems, stating that we're experiencing a 'data crisis'. In reference to [Edsger Dijkstra's Software crisis](https://en.wikipedia.org/wiki/Software_crisis) whereby software gets more complex (and more expensive) in tandem with compute power, now that software is generated by data, more and more effort needs to be spent to improving and verifying the dataset. In addition to incrased focus on data, it is also important to ensure tracability of training process and that deployed systems are constantly monitored to detect anomalous inputs and outputs.
```
Comment (drj11) [inline _posts/2019-09-27-rework-dl-summit.md]: typoe: "incrased" should be "increased"

**Review by drj11 — CHANGES_REQUESTED** [review]
Looks good. I like the comprehensive reading list.

The only substantive suggestion is to consider adding a call at the end of the article, something like "hope to see you there next year!" or "keep an eye for our own deep learning training" or something similar.

File: _posts/2019-09-27-rework-dl-summit.md (hunk 1 above)
Comment (twinkarma) [reply _posts/2019-09-27-rework-dl-summit.md:24]: Added images in next commit. Just adding normal images is fine? Do we need to add any  special metadata tags?

**Pull request merged**